server.go

```diff
@@ -795,6 +811,13 @@ func crowdRiseSignupHandler(w http.ResponseWriter, r *http.Request, email string
 		return
 	}
 
+	// avoid panics for out of bounds index access below
```

Comment:
comment captures it. Reminds me: we should add panic recover at the top level to prevent the service from crashing on panic.